tests/bearlib/languages/LanguageTest.py

```diff
@@ -21,9 +23,11 @@ def tearDown(self):
         pass
 
     def test_invalid_attribute(self):
-        with self.assertRaisesRegexp(AttributeError, 'not a valid attribute'):
+        with pytest.raises(AttributeError) as excinfo:
```

Comment:
No, the deprecation warning doesnt say to do this.

We have been intentionally removing all `pytest.raises` (https://github.com/coala/coala/issues/3808), and ideally replacing them with `assertRaisesRegex` per https://github.com/coala/coala/issues/3374 which means we can prevent using the less precise `assertRaises` (https://github.com/coala/coala/issues/3373).